frontend/src/containers/developers.tsx

```diff
@@ -183,7 +183,7 @@ const DeveloperPage = () => {
                 )}
 
                 <div className="d-flex justify-content-center mt-4 mb-3">
-                  <Button className="pl-4 pr-4" color="primary" type="button" onClick={onFilterSubmit}>
+                  <Button className="pl-4 pr-4" color="primary" type="submit" onClick={onFilterSubmit}>
```

Comment:
You can remove the onClick event on the button, to avoid triggering two events when the user will click on the button

Given you are using the onSubmit event, you should prevent page reloading on onFilterSubmit callback, with something like
```typescript
const onFilterSubmit = async (e: React.SyntheticEvent) => {
    e.preventDefault()
    //.....
};
```

Reply:
oki thank. good idea i did't think about it. i make it